html/problems.hbs

```diff
@@ -3,7 +3,7 @@
     {{#each problems}}
     <div class="card">
       <div class="card-body">
-        <h5 class="card-title">{{this.title}}</h5>
+        <a href="problems/{{this.id}}"><h5 class="card-title">{{this.title}}</h5></a>
```

Comment:
Probably want to use absolute link just to be safe

Reply:
done, ty!